life_expectancy/cleaning.py

```diff
@@ -1,47 +1,71 @@
 """
 Script to load, clean and save data
 """
-import pandas as pd
 import argparse
 import pathlib
+import pandas as pd
 
-# life_path = pathlib.Path.cwd()
-# data_path = life_path.joinpath('data')
 
-def clean_data(path: str, country: str = 'PT'):
+life_path = pathlib.Path(__file__).parent
```

Comment:
Suggestion: This can be personal (nitpicking if you want), but I would give some tweaks to the file paths and names logic, as it is a bit difficult to read and keep track of. My suggestion is for you to declare all the things related to files in here.

----- Example -----
PARENT_PATH = pathlib.Path(__file__).parent
FILE_PATH = PARENT_PATH /  'data'
INPUT_FILE_NAME = 'eu_life_expectancy_raw.tsv'
OUTPUT_FILE_NAME = 'pt_life_expectancy.csv'
INPUT_FILE_PATH = FILE_PATH / INPUT_FILE_NAME
OUTPUT_FILE_NAME = FILE_PATH / OUTPUT_FILE_NAME